lib/fuzzy-provider.coffee

```diff
@@ -1,8 +1,7 @@
 _ = require 'underscore-plus'
 Suggestion = require './suggestion'
 fuzzaldrin = require 'fuzzaldrin'
-{CompositeDisposable} = require 'event-kit'
-{TextEditor}  = require 'atom'
+{TextEditor, CompositeDisposable}  = require 'atom'
```

Comment:
They export it, so no reason not to instantiate based on that prototype. Of course, I've depended on plenty of things from atom that have then been deprecated, so YMMV :wink:.